src/main/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActor.scala

```diff
@@ -205,6 +208,21 @@ private class TaskLauncherActor(
   }
 
   private[this] def receiveTaskLaunchNotification: Receive = {
+    case InstanceOpSourceDelegate.InstanceOpRejected(op, TaskLauncherActor.OfferOperationRejectedTimeoutReason) =>
+      import scala.concurrent.ExecutionContext.Implicits.global
+      // Reschedule instance if provisioning timed out and the instance is not running yet.
```

Comment:
This is only true for the deleted instance but not for the new instance. How should we retrieve the new instance? With phase 3 we won't to worry about this.